package/Dockerfile

```diff
@@ -1,8 +1,28 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
+FROM ghcr.io/verrazzano/golang:v1.20.12 as build
+
 RUN yum update -y \
```

Comment:
Lines 3-5 can be deleted.  Should not be using yum.  Also, the golang:v1.20.12 image should be used as is, no need to do update on it.  That is what I have seen others doing.

Reply:
Sorry, I fixed it now. Thanks again.